src/main/java/de/fraunhofer/aisec/cpg/frontends/java/StatementAnalyzer.java

```diff
@@ -410,6 +444,11 @@ public CompoundStatement handleBlockStatement(Statement stmt) {
 
     if (caseExpression == null) {
       if (optionalTokenRange.isPresent()) {
+        /*
+        TODO: not sure if this is really necessary, it seems to be the same location as
+         parentLocation, except that column starts 1 character later and I am not sure if
+         this is correct anyway
+        */
         // Compute region and code for self generated default statement to match the c++ versions
```

Comment:
I am not really sure if this piece of code is really necessary. I just made it work like before but I do not really see the point in this. I have included a comment about it, so we might decide to deal with it later